### PR TITLE
china choropleth map

### PR DESCRIPTION
**Pull request by jialu-stellar-xia**
china choropleth map

### CODE REVIEW

File: china_choropleth_map.Rmd [hunk 1]
```diff
@@ -0,0 +1,267 @@
+#China choropleth map
```
Comment (jtr13) [inline china_choropleth_map.Rmd]: Please add a space after the hashtag

File: china_choropleth_map.Rmd [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+We are gonna plot total confirmed cases.
+
+## Static Map with ggplot
+Since ggplot does not have build-in China map, we download shape files from *the “Capital of Statistics” webiste* and prepare the shape data for ggplot.
+
+### Prepare the shape data of China
+```{r}
+
+dsn<-paste0(getwd(),"resources/china_choropleth_map/china-province-border-data/bou2_4p.shp")
```
Comment (jtr13) [inline china_choropleth_map.Rmd]: Please remove `getwd()` here

Comment (jtr13) [reply china_choropleth_map.Rmd]: Please make the indicated small changes and the chapter will be ready to merge. Thank you!

**Comment by jtr13** [comment]
Thank you for your contribution! 🎉 
(Please remember to submit the required links in CourseWorks to get credit.)

**Comment by jtr13** [comment]
Great chapter! A quick question: can I safely remove the `write.csv` lines? They are adding files to the root directory and it doesn't seem that they're needed as those files are not read back in.

**Comment by jialu-stellar-xia** [comment]
> Great chapter! A quick question: can I safely remove the `write.csv` lines? They are adding files to the root directory and it doesn't seem that they're needed as those files are not read back in.

Sure! You can remove that. We did this just for data backup. Sorry for the inconvenience:)

**Comment by jtr13** [comment]
No problem, thank you!

**Pull request merged**